serverengine.gemspec

```diff
@@ -34,4 +34,7 @@ Gem::Specification.new do |gem|
     # windows dependencies
     gem.add_runtime_dependency("windows-pr", ["~> 1.2.5"])
   end
+  if /linux/ =~ fake_platform || (/linux/ =~ RUBY_PLATFORM && fake_platform.empty?)
```

Comment:
Should we add `x86-linux` and `x86_64-linux` as GEM_BUILD_FAKE_PLATFORM like as follows?
https://github.com/treasure-data/serverengine/blob/master/Rakefile#L19